lib/Gtfsrt2LC.js

```diff
@@ -235,11 +235,11 @@ class Gtfsrt2LC {
     }
 
     getConnectionType(entity) {
-        if (entity.is_deleted) {
-            return 'CanceledConnection';
+        if (entity.is_deleted || entity.trip_update.trip.schedule_relationship == 3) {
+            return 'lc:CanceledConnection';
         }
         else {
-            return 'Connection';
+            return 'lc:Connection';
```

Comment:
Is not necessary to add the `lc` prefix for `Connection` because it is already defined in the `@context` of every Linked Connections page as `"Connection": "lc:Connection"`. See https://graph.irail.be/sncb/connections.

Reply:
Fixed + PR is now for the development branch